install/ossm-running-v2-same-cluster-as-v3-assembly.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+[id="ossm-running-v2-same-cluster-as-v3_{context}"]
+= Running {SMProductName} 2.6 in the same cluster as {SMProductName} 3
+include::_attributes/common-attributes.adoc[]
+:context: ossm-running-v2-same-cluster-as-v3-assembly
+
+toc::[]
+
+If you are moving from {SMProductName} v2.6, you can run {SMProduct} v2.6 side-by-side with {SMProduct} v3.0, in one cluster, without them interfering with each other.
+
+include::modules/ossm-running-v2-v3-multitenant-deployment-model.adoc[leveloffset=+1]
+include::modules/ossm-running-v2-v3-cluster-wide-deployment-model.adoc[leveloffset=+1]
+
+//For TP1 only OSSM 2.6 has been tested and verified to be able to run in same cluster as OSSM 3.0. For GA, it might other versions of 2.x. As of 10/31/2024, that is unknown, so 2.6 is specifically called out for now but the id says v2 in the event it applies to other versions of 2.x.
+
+[role="_additional-resources"]
+[id=additional-resources-v2-same-cluster-v3]
+== Additional resources
+
+* xref:../../install/ossm-installing-openshift-service-mesh.html[Installing {SMProduct}]
```

Comment:
I think the build fails because this goes up above root, try removing one `../`
```suggestion
* xref:../install/ossm-installing-openshift-service-mesh.html[Installing {SMProduct}]
```